sdcm/cluster.py

```diff
@@ -322,15 +322,42 @@ def __init__(self, name, ssh_login_info=None, base_logdir=None, node_prefix=None
         self.is_enterprise = None
         self.replacement_node_ip = None  # if node is a replacement for a dead node, store dead node private ip here
         self._set_prometheus_paths()
+        self.distro = None
```

Comment:
Current get_distro() requests instance network. If we get distro during init stage, it will wait there for a while. We expect multiple instances can't be created & setup in better parallel.